src/boundary/BoundaryData.h

```diff
@@ -33,32 +33,33 @@ enum BoundaryCondition : int {
 };
 
 class BoundaryData {
-public:
+ public:
     BoundaryData();
     ~BoundaryData();
     void print();
 
-    static std::string getFieldTypeName(FieldType f);
-    static std::string getBoundaryConditionName(BoundaryCondition bc);
-    static std::string getPatchName(Patch p);
+    static std::string get_field_type_name(FieldType f);
+    static std::string get_boundary_condition_name(BoundaryCondition bc);
+    static std::string get_patch_name(Patch p);
 
-    static FieldType matchField(const std::string& s);
-    static Patch matchPatch(const std::string& s);
-    static BoundaryCondition matchBoundaryCondition(const std::string& s);
+    static FieldType match_field(const std::string& string);
+    static Patch match_patch(const std::string& string);
+    static BoundaryCondition match_boundary_condition(const std::string& string);
 
-    void addBoundaryCondition(const std::vector<Patch>& patches, real value, BoundaryCondition boundaryCondition);
-    BoundaryCondition getBoundaryCondition(Patch p){ return m_boundaryConditions[p];};
-    real getValue(Patch p){ return m_values[p];};
+    void add_boundary_condition(const std::vector<Patch>& patches, real value, BoundaryCondition boundary_condition);
+    BoundaryCondition get_boundary_condition(Patch p) { return m_boundary_conditions[p];}
+    real get_value(Patch p) { return m_values[p];}
```

Comment:
can be const

Reply:
ded5055d03973e886d37103a145ecc5db868bd9f